Intersect.Server/Web/RestApi/Types/Player/SpellInfo.cs

```diff
@@ -0,0 +1,6 @@
+namespace Intersect.Server.Web.RestApi.Types.Player;
+
+public partial struct SpellInfo
```

Comment:
Request or response?

Also this might be better consolidated into an `IdPayload` since it's literally just a Guid Id and nothing more.